src/test/run-make-fulldeps/issue-97463-abi-param-passing/Makefile

```diff
@@ -0,0 +1,9 @@
+-include ../tools.mk
+
+all:
+#	$(call COMPILE_OBJ,$(TMPDIR)/bad.o,bad.c)
+	$(CC) -c -O3 -o $(TMPDIR)/bad.o bad.c
```

Comment:
Maybe leave a comment for why you call `CC` directly instead of with `COMPILE_OBJ`?

Reply:
Oh yeah, that was because the bug didn't replicate if I didn't pass *some* optimization flags in. I noted it my surprise about this (and my concern about what that implies for the rest of the `run-make` test suite) on the issue https://github.com/rust-lang/rust/issues/97463#issuecomment-1146308440 , but you're right, it would be good to have a note about this in the `Makefile` itself. Will fix.

Reply:
resolved via f5bbb8c187971452540888fc653485f5277309e6